cabal.project

```diff
@@ -1,3 +1,5 @@
 packages: .
 optional-packages: llvm-pretty
 constraints: directory >= 1.2.7
+-- See https://github.com/kquick/kvitable/issues/2
+allow-newer: kvitable:base
```

Comment:
I think the plan is now to merge https://github.com/kquick/kvitable/pull/4 and upload a new minor version of kvitable to Hackage. We can then delete these lines.

Reply:
Version 1.0.3.0 of kvitable is now available from Hackage.